src/code/fv-components/fv-gene-label.js

```diff
@@ -13,6 +13,10 @@ const Form = React.createClass({
     // render :: a -> ReactElement
     render: function(){
         return <SimpleSelect defaultValue={this.props.defaultOption} onValueChange={this.props.handleChange} options = {this.props.options} hideResetButton={true} editable={false} disabled={false}></SimpleSelect>;
+    },
+
```

Comment:
Previously, the dropdowns were inputs, which meant every time one was clicked, a keyboard would pop up on the iPad. I disabled the dropdowns to prevent that, but it would be nice if we could keep them enabled on desktop. Perhaps I'll add a ticket for mobile-specific behavior